lib/workers/repository/update/branch/index.spec.ts

```diff
@@ -1972,6 +1972,27 @@ describe('workers/repository/update/branch/index', () => {
       expect(commit.commitFilesToBranch).toHaveBeenCalled();
     });
 
+    it('continues when checked by checkedBranches', async () => {
+      getUpdated.getUpdatedPackageFiles.mockResolvedValueOnce({
+        ...updatedPackageFiles,
```

Comment:
why flat copy? isn't that variable reset on every test anyways?

Reply:
what's the alternative just empty like this?

```
   getUpdated.getUpdatedPackageFiles.mockResolvedValueOnce({
        updatedPackageFiles: [],
        artifactErrors: [],
        updatedArtifacts: [],
      });
```

Reply:
yes

Reply:
if i dont it will give me this, 


TS2740: Type 'PackageFilesResult' is missing the following properties from type 'FileChange[]': length, pop, push, concat, and 29 more.  get-updated.ts(18, 3): The expected type comes from property 'updatedPackageFiles' which is declared here on type 'PackageFilesResult'


i think the parameter that is received by the function has defaults and by copying the fields we add to it 3 more fields 
that is why its a flat copy